superdesk/default_settings.py

```diff
@@ -828,6 +828,12 @@ def local_to_utc_hour(hour):
 #:
 WORKFLOW_ALLOW_DUPLICATE_TO_NON_MEMBERS = False
 
+#: Allow users to copy content from desk to personal
+#:
+#: .. versionadded:: 1.34
+#:
+WORKFLOW_ALLOW_COPY_TO_PERSONAL = True
```

Comment:
Does it make sense to keep it disabled by default and enable only for ansa?

Reply:
@karelpetrak any feedback on this?

Reply:
It can be enabled by default with the option to disable it via the settings.